x/btccheckpoint/keeper/msg_server_test.go

```diff
@@ -589,3 +592,107 @@ func TestStateTransitionOfValidSubmission(t *testing.T) {
 		t.Errorf("Epoch Data missing of in unexpected state")
 	}
 }
+
+func FuzzConfirmAndDinalizeManyEpochs(f *testing.F) {
+	datagen.AddRandomSeedsToFuzzer(f, 20)
```

Comment:
```suggestion
	datagen.AddRandomSeedsToFuzzer(f, 10)
```
To follow similar tactic with other tests.

Reply:
tbh this was deliberate, as I consider this logic quite a bit important and the tests are pretty fast so there is no downside from going to 20 iterations